include/grpc/grpc.h

```diff
@@ -343,6 +344,7 @@ typedef struct grpc_op {
       int *cancelled;
     } recv_close_on_server;
   } data;
+  void *reserved;
```

Comment:
Let's move to before data, and add to the data union:

struct {
  void *reserved[8];
} reserved;